frontend/src/app/commonComponents/RadioGroup.tsx

```diff
@@ -92,27 +92,33 @@ const RadioGroup = ({
           validationStatus === "error" && "usa-form-group--error"
         )}
       >
-        {choices.map((c, i) => (
-          <div className={groupClass} key={c.value}>
-            <input
-              type="radio"
-              id={`${widgetId}_${c.value}_${i}`}
-              name={name}
-              value={c.value}
-              disabled={c.disabled || false}
-              className={inputClass}
-              checked={c.value === selectedRadio}
-              onChange={onChange}
-              {...inputProps}
-            />
-            <label
-              className="usa-radio__label"
-              htmlFor={`${widgetId}_${c.value}_${i}`}
-            >
-              {c.label}
-            </label>
-          </div>
-        ))}
+        {choices.map((c, i) => {
+          const labelClasses = classnames(
+            "usa-radio__label",
+            (c.disabled || inputProps.disabled) && "text-base"
```

Comment:
I don't know about the USWDS `text-base` but am wondering if it should have been CSS-inherited somehow rather than us having to explicitly specify it here. I wonder if we have some other class that is messing things up?

Reply:
The radios were correctly grayed out but the designers thought they were too light to read. This seemed to be like an easy-ish way to darken the text